app/views/group/index.html.erb

```diff
@@ -0,0 +1,62 @@
+<div class="login_container">
```

Comment:
- [ ] It will be great to ensure that your application runs without any errors. Although you have created the association as requested by the previous reviewer, now your application is throwing this error when I try to create a new category  ⬇️ 
![Screenshot 2023-06-11 at 09 04 03](https://github.com/hopeforkoami/Budget_App/assets/32361963/601c0bf9-9dab-4d64-9f79-965216fa4745)